backend/windmill-queue/src/jobs.rs

```diff
@@ -283,7 +283,7 @@ pub struct WrappedError {
     pub error: serde_json::Value,
 }
 
-trait ValidableJson {
+pub trait ValidableJson {
```

Comment:
Weirdly I have this error locally:
```
error[E0445]: private trait `ValidableJson` in public interface
   --> windmill-queue/src/jobs.rs:416:1
    |
286 | trait ValidableJson {
    | ------------------- `ValidableJson` declared as private
...
416 | #[instrument(level = "trace", skip_all, name = "add_completed_job")]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't leak private trait
    |
    = note: this error originates in the attribute macro `instrument` (in Nightly builds, run with -Z macro-backtrace for more info)
```
I'm not sure why, moving it to public removes the issue